src/components/bluebutton.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './bluebutton.scss';
+
+const Button = ({ targetTo, label, ...props }) =>
+(
+  <div
+    className={styles.our_button}
+    role="button"
+    tabIndex="-1"
+    {...props}
+    onClick={(event) => {
+      event.preventDefault();
+      targetTo();
+    }}
+  >
+    {label}
```

Comment:
I'ts better to use [`children`](https://facebook.github.io/react/docs/composition-vs-inheritance.html#containment) for this, than you can use the button as `<Button><img src"icon"/>Text</Button>`.